webpack.config.js

```diff
@@ -93,5 +93,8 @@ module.exports = {
       amd: 'kaltura-player-js',
       root: ['KalturaPlayer']
     }
+  },
+  optimization: {
+    minimize: PROD
```

Comment:
use `--mode production`/`--mode development` in package.json instead(for build/dev/watch scripts)